### DISCUSSION TITLE
Server automatically close sharing connection

### DISCUSSION THREAD

**Discussion by chaunnt** [rustdesk/rustdesk#5021] [Q&A]
![IMG_20230716_152110_244](https://github.com/rustdesk/rustdesk/assets/29053600/c4b76aa3-9080-4221-8c24-0fbf895efcfc)
when I'm doing remote. I see server close the connection but don't know why it happens. the server still works fine with others connections. is it limitation on resource or something ?

**Comment by 21pages** [comment]
1.1.9 or later version?

**Reply by chaunnt** [reply]
1.1.9 sorry.
